collections/eservice-template/Update EService Template.bru

```diff
@@ -26,6 +26,6 @@ body:json {
     "description": "This is the description of the EService",
     "technology": "REST",
     "mode": "DELIVER",
-    "isSignalHubEnabled": false
+    "isSignalHubEnabled": true
```

Comment:
why this changes?

Reply:
It's just to have by default a change from false to true when updating, since this is set to false in the original create call